mirrord-protocol/src/tcp.rs

```diff
@@ -38,3 +38,12 @@ pub enum DaemonTcp {
     Data(TcpData),
     Close(TcpClose),
 }
+
+/// Messages related to Steal Tcp handler from client.
+#[derive(Encode, Decode, Debug, PartialEq, Eq, Clone)]
+pub enum LayerStealTcp {
```

Comment:
Consistency, the `DaemonMessage` is `TcpSteal`, so change this to `LayerTcpSteal`.